bigbluebutton/bigbluebutton
# feat(plugin): add setDisplayActionsBar ui-command

### What does this PR do?

It adds the possibility to display or hide the actions-bar via UI-command.


### How to test

One can use the SampleActionButtonDropdown in the plugin-SDK to see this feature working, just like in the demo! (See **More** section)

### More

Closely related to https://github.com/bigbluebutton/bigbluebutton-html-plugin-sdk/pull/137

See demo below (just a minor explanation: If the user clicks the button, the plugin sends a ui-command to first close the actions-bar, and then, after 5 seconds, it sends a ui-command again to open it since one can't reach that button again. That's why it  automatically returns, the developer has full control over it).


https://github.com/user-attachments/assets/7cdad3a1-b919-4f30-8e3b-9ad265e0ef85
